easybuild/framework/easyblock.py

```diff
@@ -1351,10 +1352,16 @@ def make_module_extend_modpath(self):
             # add user-specific module path; use statement will be guarded so no need to create the directories
             user_modpath = build_option('subdir_user_modules')
             if user_modpath:
+                user_envvars = build_option('envvars_user_modules') or [DEFAULT_ENVVAR_USERS_MODULES]
                 user_modpath_exts = ActiveMNS().det_user_modpath_extensions(self.cfg)
                 self.log.debug("Including user module path extensions returned by naming scheme: %s", user_modpath_exts)
-                txt += self.module_generator.use(user_modpath_exts, prefix=self.module_generator.getenv_cmd('HOME'),
-                                                 guarded=True, user_modpath=user_modpath)
+                for user_envvar in user_envvars:
+                    if not os.getenv(user_envvar):
+                        self.log.warning("Requested environment variable $%s as an additional branch for user "
```

Comment:
Not even sure we even need this warning any more since I can now handle the case where the environment variable is not defined (for both Tcl and Lua modules)